### ISSUE TITLE
vscode: abilility to detect python interpreter

### ISSUE THREAD

**Issue by DetachHead** [robocorp/robotframework-lsp#416]
**Is your feature request related to a problem? Please describe.**
currently, you have to manually specify your python interpreter.

`.vscode/settings.json`:
```json
{
    "robot.language-server.python": "${workspaceFolder}/.venv/bin/python"
}
```

this isn't ideal, because if i want to commit the settings it's hardcoded to where python is located on my machine.

**Describe the solution you'd like**
it should be able to automatically detect the python path, like the mypy and flake8 linters do:
![image](https://user-images.githubusercontent.com/57028336/129120981-23bec77b-ed15-4c5f-93e9-efb16731a959.png)

**Describe alternatives you've considered**
i tried this:
```json
{
    "robot.language-server.python": "${workspaceFolder}${env:VIRTUAL_ENV}/python"
}
```
but that didn't work, not sure why. (it also assumes that everyone is using a virtualenv which also isn't ideal)

**Comment by DetachHead** [comment]
looks like `${command:python.interpreterPath}` does what i want, but for some reason robot can't resolve it even though it works everywhere else
![image](https://user-images.githubusercontent.com/57028336/133735716-a29af183-be40-4662-a58c-e5a82689f4de.png)

tasks.json:
```json
{
    "version": "2.0.0",
    "tasks": [
        {
            "label": "mypy",
            "type": "shell",
            "command": "${command:python.interpreterPath} -m mypy ."
        }
    ]
}
```
![image](https://user-images.githubusercontent.com/57028336/133735972-114b3d06-e8f0-4014-87fc-f9b830b1775e.png)

**Comment by DetachHead** [comment]
i guess because vscode doesn't expose the api for resolving variables, lame https://github.com/microsoft/vscode/issues/46471

**Comment by podfran** [comment]
I have a similar issue in PyCharm. The default is to search in the PATH, when it is very common to use a virtual env (PyCharm asks about this automatically). I need to set the path to the python executable for each new project. Maybe adding macros to the global settings would be a solution?

**Comment by fabioz** [comment]
@podfran please create a separate issue for PyCharm.

I'm closing this one as this was actually implemented in the meanwhile...

**Comment by fabioz** [comment]
p.s.: according to the changelog (https://github.com/robocorp/robotframework-lsp/blob/master/robotframework-ls/docs/changelog.md, this was implemented in `0.38.0 (2022-01-26)`).
